en/components/grids_templates/summaries.md

```diff
@@ -534,7 +432,115 @@ When a default summary is defined, the height of the summary area is calculated
 
 }
 
+## Disable Summaries
+
+The [`disabledSummaries`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#disabledSummaries) property provides precise control over the summary functions displayed for each column in the grid. This property is particularly useful when you want to remove unnecessary or less relevant summaries, ensuring that only meaningful and context-specific results are presented to users. It accepts an array of strings, such as `['count', 'min', 'max']`, where each string corresponds to a **summary key** that should be excluded from the column's displayed summaries.
 
```

Comment:
If these empty lines can be omitted and still have the paragraph starting on a new line - that would be great.
![image](https://github.com/user-attachments/assets/0d719db2-48b0-4e7b-8150-bf25724ed576)